### PR TITLE
fix(Harvest): Better diffentiate bankId

### PR DESCRIPTION
**Pull request by doubleface**
The linxo bankId is the one saved into the account.
I renamed linxo bankId to linxoBankId et BI bankId to BiBankId

### CODE REVIEW

File: packages/cozy-harvest-lib/src/services/budget-insight.spec.js [hunk 1]
```diff
@@ -368,7 +368,7 @@ describe('handleOAuthAccount', () => {
       t,
       account: {
         ...account,
-        ...{ auth: { bankId: TEST_BANK_BI_ID } },
+        ...{ auth: { bankId: TEST_BANK_COZY_ID } },
```
Comment (ptbrowne) [inline packages/cozy-harvest-lib/src/services/budget-insight.spec.js:371]: TEST_BANK_LINXO_ID ?

**Review by ptbrowne — COMMENTED** [review]
It's a bit weird to see linxo coming back here, I wonder if we should not name it the cozyBankId ?

File: packages/cozy-harvest-lib/src/services/budget-insight.spec.js (hunk 1 above)
Comment (doubleface) [reply packages/cozy-harvest-lib/src/services/budget-insight.spec.js:371]: I actually prefer to replace all references to Linxo by Cozy

**Review by ptbrowne — APPROVED** [review]
OK we still have the get getMappingLinxoBI leaking a bit our history with Linxo, but this can be renamed later if necessary. LGTM 👍

**Pull request merged**